### PR TITLE
init

### PR DESCRIPTION
**Pull request by d-smyrnov**

### CODE REVIEW

File: Safor/index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                <a href="#"><img src="img/head/logo.svg"></a>
+                <ul>
+                    <a href="#"><li>How it works</li></a>
+                    <a href="#"><li>About</li></a>
+                    <a href="#"><li>Contacts</li></a>
+                </ul>
+            </div>
+            
+            <div class="one">
+                <h1>SAFOR</h2>
```
Comment (ghost) [inline Safor/index.html]: These two h1 should be represented as one.

File: Safor/index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                <div class="two_video">
+                    <h1>WHAT IS SAFOR?</h1>
+                    <a href="#">
+                        <img src="img/two/Rectangle.svg">
+                        <p> WATCH NOW</p>
+                    </a>
+                </div>
+        
+                <ul class="two_text">
+                    <li>
```
Comment (ghost) [inline Safor/index.html:121]: Nesting p into li makes no sense

File: Safor/index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            <h2>Complete the registration</h2>
+            <h2>SECURE YOUR LIFE IN ONE TAP</h2>
+            <p>Additional</p>
+            <p>Set up your home and office locations to simplify your search in the city.
+                Add your family contact to notify them in dangerius situation</p>
+        </div>
+    
+    </div>
+
+    <div class="four">
```
Comment (ghost) [inline Safor/index.html]: replace div for each section with section tag

File: Safor/index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        <hr class="three_background">
+        <img class="three_background" src="img/three/Bitmap1.png">
+        <img class="three_background" src="img/three/Bitmap2.png">
+
+        <div class="three_text">
+            <p>HOW IT WORKS</p>
+            <h2>Install app IOS / ANDROID</h2>
+            <h2>Complete the registration</h2>
+            <h2>SECURE YOUR LIFE IN ONE TAP</h2>
+            <p>Additional</p>
```
Comment (ghost) [inline Safor/index.html]: Margin from top item looks incorrect on 1000px resolution

File: Safor/index.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    <title>Document</title>
+</head>
+<body>
+
+    <div class="topLayer">
+
+            <div class="head">
+                <a href="#"><img src="img/head/logo.svg"></a>
+                <ul>
+                    <a href="#"><li>How it works</li></a>
```
Comment (ghost) [inline Safor/index.html]: Incorrect font-size on 759px resolution

**Review by ghost — CHANGES_REQUESTED** [review]

File: Safor/index.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+                            our work and communication.</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="five_two">
+                <h2 class="five_two_headline">FOR PEOPLE</h2>
+                <ul>
+                    <li>
+                        <img src="img/five/Polygon.png">
+                        <p>Do not waste time to explain your location and nature of the threat, guard
```
Comment (ghost) [inline Safor/index.html:123]: On 1024px i can see text from white block, when grey is active. Please fix.

File: Safor/index.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+                        <p>And we will help in any situation. Only after coming to you the calling
+                            will be closed by the patrol arrived, whose goal is to protect and help you.</p>                    
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="six">
+            <h1 class="six_headline">HAVE QUESTIONS? CONTACT US</h1>
+            <form class="six_inputs" action="#">
+                <input type="text" placeholder="NAME">
```
Comment (ghost) [inline Safor/index.html:144]: Distance between placeholder and bottom border should be as on mockup

File: Safor/index.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+                <input type="text" placeholder="MESSAGE">
+                <input type="submit" value="SEND">
+            </form>
+        </section>
+        <section class="footer">
+            <img src="img/footer/Group.svg">
+            <p>© SAFOR 2018 All Rights Resevered 85 Great Portland Street, First Floor London, W1W7LT, United Kingdom</p>
+            <div>
+                <a href="#"><img src="img/footer/1.svg"></a>
+                <a href="#"><img src="img/footer/2.svg"></a>
```
Comment (ghost) [inline Safor/index.html:155]: On resolution 375, size of instagram icon incorrect. Also check distance between icons.

File: Safor/index.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+                    <h3 class="one_text">Safor is an intelligent personal guard platform that empowers individuals to enjoy active and safe lives, while 
+                        their friends and family can enjoy peace of mind.</h3>
+                    <div class="one_background">
+                        <hr>
+                        <hr>
+                        <hr>
+                        <hr>
+                        <hr>
+                        <hr>
+                        <hr>
```
Comment (ghost) [inline Safor/index.html:32]: On resolution 320 these lines should be 1px, but now they are 2px

**Review by ghost — CHANGES_REQUESTED** [review]

File: Safor/index.html (hunk 8 above)
Comment (ghost) [reply Safor/index.html:155]: Try to use another instagram and maybe another social icons on lower resolution.

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]

File: Safor/index.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Document</title>
+    </head>
+    <body>
+        <section class="topLayer">
```
Comment (volo-maslow) [inline Safor/index.html]: Strange naming of the section, please use dashes or underscores to separate different words.

File: Safor/index.html [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+                    </li>
+                    <li>
+                        <img src="img/five/Rectangle_6_Copy_2.png">
+                        <p>And we will help in any situation. Only after coming to you the calling
+                            will be closed by the patrol arrived, whose goal is to protect and help you.</p>                    
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="six">
```
Comment (volo-maslow) [inline Safor/index.html]: Change names of sections to be more descriptive, for example: footer-feedback or feedback-form.
Please do this for every section and sub div in the document.

**Review by volo-maslow — CHANGES_REQUESTED** [review]

**Review by volo-maslow — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**